enhancements/authentication/AuthConfig-missing-fields.md

```diff
@@ -0,0 +1,503 @@
+- This approach is subject to change depending on discussions regarding HyperShift and Standalone OpenShift's unified feature maturity. For now, we will proceed with expectations for standalone OpenShift.
+  
+To graduate from Tech Preview (TP) to GA, the following criteria must be met:
+
+- User-facing documentation exists.
+- Time for early adopter feedback in Tech Preview.
+- Telemetry/Metrics are in place.
+- Additional testing will be conducted based on early adopter and anticipated user feedback, particularly for desired configurations.
+
+### Removing a deprecated feature
+
+N/A
+
+## Upgrade / Downgrade Strategy
+
+## Upgrade Considerations
+
+When upgrading from a version of OpenShift that does not include these API changes to a version that does, the upgrade process should be seamless. The new API fields will be introduced, but they will not affect existing configurations unless explicitly used. No manual intervention is required before the upgrade, and clusters should continue functioning as expected without any disruptions.
+After upgrading, customers can begin leveraging the new API fields by updating the Authentication resource to configure token claim mappings, validation rules, and user validation settings.
+
+## Downgrade Considerations
+
+If a customer downgrades from a version of OpenShift where these API changes are present to a version where they are not, any configurations using the new fields will no longer be recognized. This could lead to unexpected behavior or validation errors. To ensure a smooth downgrade, customers should remove any references to the new API fields before initiating the downgrade process.
+
+## Version Skew Strategy
+
+Since this enhancement builds upon the existing OIDC functionality, it will follow the same [version skew strategy established for the original OIDC feature ](https://github.com/openshift/enhancements/blob/master/enhancements/authentication/direct-external-oidc-provider.md#version-skew-strategy). These changes do not introduce any new version skew concerns beyond those already considered in the initial implementation of OIDC support. 
+As a result, there are no additional compatibility risks or upgrade constraints beyond what has already been accounted for in the existing OIDC version skew strategy.
+
+## Operational Aspects of API Extensions
+
+## Impact of Misconfiguration on Users
+
+If users configure the new API fields incorrectly, authentication may break, causing the kube-apiserver to become inaccessible. This could prevent users from logging in or interacting with the cluster.
+
+## Mitigation Strategy
+
+We plan to add a combination of admission time and runtime checks to prevent invalid configurations from being applied. These checks will ensure that only properly formatted and functional configurations are accepted before they are rolled out to the kube-apiserver.
```

Comment:
It would make sense to include that there if the external OIDC authentication layer is broken, using a kubeconfig should still function correctly as a break-glass scenario.